tests/functional/adapter/test_simple_snapshot.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from dbt.tests.adapter.simple_snapshot.test_snapshot import (
+    BaseSimpleSnapshot,
+    BaseSnapshotCheck,
+)
+from dbt.tests.util import run_dbt
+
+
+@pytest.mark.delta
```

Comment:
Snapshots are not supported on memory connector:
```
io.trino.spi.TrinoException: This connector does not support modifying table rows
```

(all [current snapshot tests](https://github.com/starburstdata/dbt-trino/blob/master/tests/functional/adapter/materialization/test_snapshot.py) are executed on iceberg or delta)


I run them on delta, because on Iceberg I receive:
```
Timestamp precision (3) not supported for Iceberg. Use "timestamp(6) with time zone" instead.
```

Should I adjust and run this tests on both Iceberg and Delta?

Reply:
Yes please adjust both tests

Reply:
Adjusted in `test_simple_snapshot.py`. Can't add tests in `test_simple_copy.py` for Delta, as Delta don't support materialized views.